### PR TITLE
Nandorfs/plugin should i deploy

### PR DESCRIPTION
**Pull request by NandoRFS**
*New friday! New Plugin!* 🎉

## Introducing a new "Should I Deploy?" Plugin!

Who has never thought for hours about whether or not to deploy on a certain day of the week? If you've been through this, you certainly understand me! No more accessing the calendar and worrying about the deployment day, go into your backstage and get the information right away!

Now you have a complete card! Should I deploy today? IT'S FRIDAY! PLEASE DON'T DO IT! A little joke never hurt anyone, especially a useful joke like this!

should-i-deploy-card-without-title

## Future Ideas

- Block deployment actions? crazy! 😎
- Personalized jokes? Wonderful. 🙀
- Maybe a popup about deployment actions, are you sure? it's Friday! LOL 😈

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by tudi2d — CHANGES_REQUESTED** [review]
funny idea, @NandoRFS - hope you had fun developing it as well! :) Heads-up that we have an open RFC at the moment discussing to stop accepting plugins in the main repository (https://github.com/backstage/backstage/issues/20266). 

Would you be up to move this plugin out to a separate repository & just add the metadata to the `microsite` in this PR? This way people can still discover it through the public Plugin Directory.

**Comment by NandoRFS** [comment]
Hi @tudi2d! of course i had a fun development moment 😄, its my first time contributing 👀 , i'll change ASAP to suggested model. Thanks for fast response, I apreciate it!

**Comment by NandoRFS** [comment]
Hey Guys! I move the plugin to my personal repository in commit [f24792c](https://github.com/backstage/backstage/pull/20975/commits/f24792ca746b7f63ebb2b7a7fc9bb7a1d7549b20) and published the package on NPM, leaving only microsite data. @tudi2d I think everything is fine now, awaiting for new review 😄

File: microsite/data/plugins/should-i-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Should I Deploy Today?
+author: NandoRFS
+authorUrl: https://github.com/NandoRFS
+category: Humor
```
Comment (tudi2d) [inline microsite/data/plugins/should-i-deploy.yaml:5]: TIL this category already exists - that's great :D

**Review by tudi2d — APPROVED** [review]

File: microsite/data/plugins/should-i-deploy.yaml (hunk 1 above)
Comment (NandoRFS) [reply microsite/data/plugins/should-i-deploy.yaml:5]: Yes, in the company where I work we already use the plugin that is in this category 😄 . Just one more question, when will the changes be merged? 🤔

**Comment by NandoRFS** [comment]
Hey Guy! @tudi2d needs something else to be able to merge? 🤔

**Review by freben — APPROVED** [review]
Took the liberty of removing the changeset that was no longer needed.

**Pull request merged**